src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1017,6 +1017,14 @@ Made with 💗 by Microsoft and the PowerToys community.</value>
   <data name="FileExplorerPreview_RebootRequired.Title" xml:space="preserve">
     <value>A reboot may be required for changes to these settings to take effect</value>
   </data>
+  <data name="FileExplorerPreview_PreviewHandlerOutlookIncompatibility.Title" xml:space="preserve">
+    <value>Enabling the preview handlers will override other preview handlers already installed - there have been reports of incompatibility between Outlook and the PDF Preview Handler</value>
```

Comment:
```suggestion
    <value>Enabling the preview handlers will override other preview handlers already installed. There have been reports of incompatibility between Outlook and the PDF Preview Handler.</value>
```

Reply:
There's some inconsistency regarding the use of punctuation in different settings pages. Since the other existing info box in this screen also doesn't have punctuation, I ended up not using punctuation for these as well.